pyvisa/highlevel.py

```diff
@@ -1450,14 +1450,20 @@ def list_backends():
 def get_wrapper_class(backend_name):
     """Return the WRAPPER_CLASS for a given backend.
 
+    backend_name == 'ni' is used for backwards compatibility
+
     :rtype: pyvisa.highlevel.VisaLibraryBase
     """
     try:
         return _WRAPPERS[backend_name]
     except KeyError:
-        if backend_name == 'ni':
+        if backend_name == 'ivi':
```

Comment:
This code can be simplified by using `or` in the test to avoid duplicating the body of the if. Additionally it would be nice to emit a `FutureWarning` if `ni` is used.